### PR TITLE
Repoclosure exit with 1 if unsatisfied dependencies

### PR DESCRIPTION
**Pull request by jrohel**
If dnf repoclosure finds unsatisfied dependencies in a repo then it will exit with 1.
For better usage with scripts and yum repoclosure compatibility.

### CODE REVIEW

File: plugins/repoclosure.py [hunk 1]
```diff
@@ -66,6 +66,8 @@ def run(self):
             print("  unresolved deps:")
             for dep in unresolved[pkg]:
                 print("    {}".format(dep))
+        if len(unresolved) > 0:
+            self.cli.demands.success_exit_status = 1
```
Comment (ignatenkobrain) [inline plugins/repoclosure.py]: shouldn't we just `return 1` from here?

Comment (jrohel) [inline plugins/repoclosure.py]: return 1 was my first idea. But dnf works another way. Return code of run func is not used.

Comment (ignatenkobrain) [inline plugins/repoclosure.py]: what about raise dnf.exceptions.XXXX() then or something like that?

**Comment by jrohel** [comment]
retest this please

**Comment by jrohel** [comment]
retest this please

**Comment by jrohel** [comment]
retest this please

**Comment by jrohel** [comment]
retest this please

**Comment by jrohel** [comment]
retest this please

**Comment by jrohel** [comment]
retest this please

File: plugins/repoclosure.py (hunk 1 above)
Comment (jrohel) [inline plugins/repoclosure.py]: I changed the code. Exception is now used.

**Comment by jrohel** [comment]
retest this please

**Comment by jrohel** [comment]
Unit tests were updated.

**Comment by ignatenkobrain** [comment]
This plugin now lives in dnf-plugins-core, please resend PR against it.

**Comment by jrohel** [comment]
OK. I sent PR to  dnf-plugins-core.

**Pull request closed without merging**